composer.lock

```diff
@@ -4,8 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "hash": "ebe771ebefde29ab37094e27b1a8383d",
```

Comment:
@slde-flash After a bit of research, I definitely can say the composer.lock is right to be changed, since as previously said that `hash` attribute is the hash of the full composer.json, so any change in the composer.json file should be followed by changes in the composer.lock. That's something we missed for example in #95. If there is no new package added to the `require` or `require-dev` which would force an update of the composer.lock, then it would be necessary to update the composer.lock file executing `composer update --lock` in order to update the composer.lock.